blog.Rmd

```diff
@@ -22,68 +22,131 @@ opts_chunk$set(
 
-If the drainage basin areas for paired water quality and discharge sites are different (in the site metadata file, e.g `siteInfo.csv`), discharge is scaled by the appropriate ratio.  The resulting estimates descibe fluxes at the water quality monitoring site.  
+All the site metadata is stored in a `loadflex::metadata` R object, which is referenced throughout the script.
 
-## Output
-Like the input files, the output files are written to a separate folder for each constituent. Output contains 5 files, which cover all the sites for that consituent: 
+The script uses several built-in `loadflex` functions to create the outputs in R, which the script then writes to files. Those functions include `summarizeInputs`, `summarizeModel`, `predictSolute`, and `aggregateSolute`. Descriptions of these functions can be found by typing `?` followed by the function name at the R prompt.
 
-* four .csvs:
-    * input data summary
-    * annual predicted loads for each model
-    * multiyear predicted loads for each model
-    * model metrics 
-    
-* PDF of plots of input data, predicted loads, and model diagnostics
+## Output files
+
+Like the input files, the output files are written to a separate folder for each constituent. Most important are the five summary files, each of which covers all the sites for that consituent. These are prefixed with the constituent name, so for NO3, these are:
+
+```
+- output
+  - NO3
+    - NO3_inputs.csv       # input data summary
+    - NO3_annual.csv       # annual predicted loads from each model
+    - NO3_multiYear.csv    # multiyear predicted loads from each model
+    - NO3_modelMetrics.csv # diagnostics and descriptive metrics for each model
+    - NO3_plots.pdf        # plots of input data, predicted loads, and model diagnostics
+```
       
-Additionally, there are four folders corresponding to the four csvs that contain individual .csv files for each site.
+Additionally, there are four folders containing individual .csv files for each site. The four .csv files listed above combine the contents of those folders.
 
 ## Conventions to remember
+
 * Column names and metadata slots for site information are assumed to refer to the water quality site, unless the name specifies they refer to discharge.
 
-* Constituent and discharge symbols need to be consistent throughout, including directory names and in the site metadata csv.
+* IDs for constituents, discharge, and sites need to be consistent across the user inputs file, the `siteInfo` file, and the folder and file names in the `siteInputs` folder.
+
+## What's next?
+
+Create your own input files following the conventions above. Edit that one line of `batch.R` to point to your own user inputs YAML file. Then run `source('batch.R')` to produce outputs for your sites.
```

Comment:
I'm glad you're holding me accountable. I wrote that, thought I should do better but didn't see how, and then left it alone like a lazy bum. You're right - I'll think of something.